src/pages/pricing.astro

```diff
@@ -26,12 +26,12 @@ const metadata = {
   <Prices
     title="Our prices"
     subtitle="Only pay for what you need"
-    prices={[
+    impactFactors={[
```

Comment:
Your remarks are correct. But at the moment we are not using the pricing page and are not planning to use it. It will be removed in the future commits. Thank you for the remark.

Redundant